tools/src/bin/benchmark-clvm-cost.rs

```diff
@@ -588,6 +678,18 @@ pub fn main() {
                 "per-byte",
             );
         }
+        if (op.flags & LIST_LENGTH_COST) != 0 {
+            write_gnuplot_header(&mut *gnuplot, op, "per-pair", "num pairs");
+            let mut output = maybe_open(options.plot, op.name, "per-pair.log");
+            let (slope, intercept): (f64, f64) = time_per_cons_for_list(&mut a, &mut output);
+            let cost = slope * cost_scale;
+
+            println!("   time: per-pair: {:.2}ns", slope);
+            println!("   cost: per-pair: {:.0}", cost);
```

Comment:
```suggestion
            println!("   time: per-node: {:.2}ns", slope);
            println!("   cost: per-node: {:.0}", cost);
```
You call this `PER_NODE` in the cost constant